server/masked_paths.go

```diff
@@ -0,0 +1,88 @@
+package server
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"slices"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/containers/common/pkg/config"
+)
+
+// appendDefaultMaskedPaths is retrieving the default masked paths and appends
+// the existing ones to it.
+func appendDefaultMaskedPaths(additionalPaths []string) []string {
+	paths := slices.Concat(defaultLinuxMaskedPaths(), additionalPaths)
+	slices.Sort(paths)
+
+	return slices.Compact(paths)
+}
+
+// defaultLinuxMaskedPaths will be used to evaluate the default masked paths once.
+var defaultLinuxMaskedPaths = sync.OnceValue(func() []string {
+	maskedPaths := slices.Concat(
+		config.DefaultMaskedPaths,
+		[]string{"/proc/asound", "/proc/interrupts"},
```

Comment:
can we add these to `"github.com/containers/common/pkg/config"` so we don't need to maintain a separate list?

We can move the entire logic there, even the `possibleCPUs` function if you prefer it to using Glob